Packs/EWS/Integrations/EWSO365/EWSO365.py

```diff
@@ -1588,6 +1603,321 @@ def mark_item_as_read(
+
+            attachments.append({
+                'name': filename,
+                'data': data,
+                'cid': cid
+            })
+
+        except Exception as e:
+            demisto.error(f'Invalid entry {file_id} with exception: {e}')
+            return_error(f'Entry {file_id} is not valid or is not a file entry')
+
+    return attachments
+
+
+def handle_transient_files(transient_files, transient_files_contents, transient_files_cids):
+    """Creates the transient attachments data
+
+    Args:
+        transient_files (str): String representation of the transient files names list.
+        transient_files_contents (str): String representation of the transient files content list.
+        transient_files_cids (str): String representation of the transient files content ids list.
+
+    Returns:
+        List[Dict]. List of the transient files data.
+    """
+    transient_attachments = []
+
+    files_names = argToList(transient_files)
+    files_contents = argToList(transient_files_contents)
+    files_cids = argToList(transient_files_cids)
+
+    for index in range(len(files_names)):
+        file_name = files_names[index]
+
+        if index >= len(files_contents):
+            break
+
+        file_content = bytes(files_contents[index], UTF_8)
+
+        if index >= len(files_cids):
+            file_cid = ''
+        else:
+            file_cid = files_cids[index]
+
+        transient_attachments.append({
+            'name': file_name,
+            'data': file_content,
+            'cid': file_cid
+        })
+
+    return transient_attachments
+
+
+def handle_template_params(template_params):
+    """Translates the template params if they exist from the context
+
+    Args:
+        template_params (str): JSON string that represent the variables names to be replaced and the desired value.
+                                Value can be either real value or context key to fetch the value from.
+
+    Returns:
+        Dict. `variable_name: value_to_use` of the templated parameters.
+    """
+    actual_params = {}
+
+    if template_params:
+        try:
+            params = json.loads(template_params)
+
+            for p in params:
+                if params[p].get('value'):
+                    actual_params[p] = params[p]['value']
+                elif params[p].get('key'):
+                    actual_params[p] = demisto.dt(demisto.context(), params[p]['key'])
+        except ValueError as e:
+            return_error('Unable to parse template_params: %s' % (str(e)))
+
+    return actual_params
+
+
+def create_message_object(to, cc, bcc, subject, body, additional_headers):
+    """Creates the message object according to the existence of additional custom headers.
+    """
+    if additional_headers:
+        return Message(
+            to_recipients=to,
+            cc_recipients=cc,
+            bcc_recipients=bcc,
+            subject=subject,
+            body=body,
+            **additional_headers
```

Comment:
why not to default the additional headers as empty string

Reply:
I tried using it as an empty dict, but it refuses to accept it